src/code/qualification/plotQualificationComparisonTimeProfile.m

```diff
@@ -38,7 +38,7 @@ function plotQualificationComparisonTimeProfile(WSettings,figureHandle,CompTimeP
     [csvSimFile, xmlfile] = getSimFile(Curves{i}, SimulationMappings, REInputPath);
     if isempty(csvSimFile)
         ME = MException('plotQualificationComparisonTimeProfile:notFoundInPath', ...
-            'In Comparison Time Profile plot %d, Mapping %d, Project "%s" or Simulation "%s" were not found in SimulationMappings', figureHandle, i, Curves{i}.Project, Curves{i}.Simulation);
+            'In Mapping %d, Project "%s" or Simulation "%s" were not found in SimulationMappings', i, Curves{i}.Project, Curves{i}.Simulation);

```

Comment:
@pchelle Why did u remove "Comparison time profile N" from the error message? It is really helpfull wenn looking for errors!

Reply:
Yes, I will add these update asap.
There was another reason why I removed it: I noticed that the plot number is actually already displayed by the the try/catch of runQualificationWorkflow, consequently the plot Index information is redundant.